lib/pnode/node.mli

```diff
@@ -14,7 +14,7 @@ val init : ?init_peers:Address.t list -> Address.t -> t ref Lwt.t
 
 val run_server :
   ?preprocessor:(Message.t -> Message.t) ->
-  msg_handler:(Message.t -> bytes option) ->
+  msg_handler:(Message.t -> bytes * bytes option) ->
```

Comment:
Should be `Message.t -> (bytes * bytes option) option` which is kind of visually confusing so maybe we should make a record type?

Reply:
Done, also re-used it into `Message.t`